src/component/NMRium.tsx

```diff
@@ -217,6 +220,10 @@ function InnerNMRium({
     }
   }, [onDataChange, state]);
 
+  useEffect(() => {
+    onViewChange?.(state.view);
+  }, [onViewChange, state.view]);
```

Comment:
You can remove the `onViewChange` dependency and put it in a ref instead.

If the api consumer redeclares the callback on every render, we don't want the callback to be called without any view state change.